content/doc/tutorials/create-a-pipeline-in-blue-ocean.adoc

```diff
@@ -8,430 +8,294 @@ section: doc
-. In the *Name* and *Value* fields that appear, specify `CI` and `true`,
-  respectively.
+image:tutorials/pipeline-in-blueocean-22-add-test-stage.png[alt="Add 'Test' stage",width=45%]
+. In this panel, type `Test` in the *Name your stage* field and then select the *Add Step* button below to open the *Choose step type* panel.
+. In this panel, select *Shell Script* near the top of the list.
+. In the resulting *Test / Shell Script* panel, specify `./jenkins/scripts/test.sh` and then select the top-left back arrow icon image:tutorials/pipeline-in-blueocean-16a-return-from-step-icon.png[alt="Return from step icon",width=3%] to return to the Pipeline stage editor.
++
+. At the lower-right of the panel, select *Settings* to reveal this section of
+the panel.
+. Select the *+* icon at the right of the *Environment* heading where you'll configure an environment directive.
+. In the *Name* and *Value* fields that appear, specify `CI` and `true`, respectively.
++
 [.boxshadow]
-image:tutorials/pipeline-in-blueocean-23-environment-directive.png[alt="Environment
-directive",width=50%] +
-  *Note:* For an explanation of this directive and its step, refer to
-  annotations *1* and *3* of the Declarative Pipeline in the
-  link:../build-a-node-js-and-react-app-with-npm/#add-a-test-stage-to-your-pipeline[``Add a test stage...`` section of the Build a Node.js and React app]
-  tutorial.
-. ( _Optional_ ) Click the top-left back arrow icon
-  image:tutorials/pipeline-in-blueocean-16a-return-from-step-icon.png[alt="Return
-  from step icon",width=3%] to return to the main Pipeline editor.
-. Click the *Save* button at the top right to begin saving your Pipeline with
-  with its new "Test" stage.
-. In the *Save Pipeline* dialog box, specify the commit message in the
-  *Description* field (e.g. `Add 'Test' stage`).
-. Leaving all other options as is, click *Save & run* and Jenkins proceeds to
-  build your amended Pipeline.
-. When the main Blue Ocean interface appears, click the _top_ row to see Jenkins
-  build your Pipeline project. +
-  *Note:* You'll notice from this run that Jenkins no longer needs to download
-  the Node Docker image. Instead, Jenkins only needs to run a new container from
-  the Node image downloaded previously. Therefore, running your Pipeline this
-  subsequent time should be much faster. +
-  If your amended Pipeline ran successfully, here's what the Blue Ocean
-  interface should look like. Notice the additional "Test" stage. You can click
-  on the previous "Build" stage circle to access the output from that stage.
+image:tutorials/pipeline-in-blueocean-23-environment-directive.png[alt="Environment directive",width=50%]
++
+NOTE: For an explanation of this directive and its step, refer to annotations *1* and *3* of the Declarative Pipeline in the link:../build-a-node-js-and-react-app-with-npm/#add-a-test-stage-to-your-pipeline[``Add a test stage...`` section of the Build a Node.js and React app] tutorial.
```

Comment:
The quotes are not displayed around the "section" text like it is in the other examples. I am not sure why it isn't rendering the same. Maybe because there is an extra space? (In case you want to display the quotes. And for consistency.)